docs/SUMMARY.md

```diff
@@ -193,6 +193,7 @@
   * [Templates](contributing-to-airbyte/templates/README.md)
     * [Connector Doc Template](contributing-to-airbyte/templates/integration-documentation-template.md)
 * [Understanding Airbyte](understanding-airbyte/README.md)
+  * [Glossary of Terms](understanding-airbyte/glossary.md)
```

Comment:
this should probably be at the bottom of the list since it's more of a reference rather than something that people read end-to-end